AbletonMCP_Remote_Script/handlers/devices.py

```
@@ -0,0 +1,1360 @@
+                sample.texture_grain_size = float(texture_grain_size)
+                changes["texture_grain_size"] = float(texture_grain_size)
+            if texture_flux is not None:
+                sample.texture_flux = float(texture_flux)
+                changes["texture_flux"] = float(texture_flux)
+            if tones_grain_size is not None:
+                sample.tones_grain_size = float(tones_grain_size)
+                changes["tones_grain_size"] = float(tones_grain_size)
+        changes["device_name"] = device.name
+        changes["track_index"] = track_index
+        changes["device_index"] = device_index
+        return changes
```

Ruff (0.15.0)

[warning] Consider moving this statement to an `else` block

(TRY300)